pkg/qe-tests/cypress/integration/models/providerVmware.ts

```diff
@@ -153,9 +153,16 @@ export class ProviderVmware extends Provider {
     this.populate(providerData);
   }
 
-  edit(providerData: VmwareProviderData): void {
+  edit(providerData: VmwareProviderData, incorrectProviderData: VmwareProviderData): void {
```

Comment:
You shouldn't have additional provider data sent here, each time all fields are updated with the new name. Please remain sending only one provider data, if you want to put incorrect inputs on purpose - just do that.

Reply:
Thanks @ibragins, Please review !